src/passes/pass.cpp

```diff
@@ -129,8 +129,11 @@ void PassRegistry::registerPasses() {
   registerPass("extract-function",
                "leaves just one function (useful for debugging)",
                createExtractFunctionPass);
-  registerPass(
-    "flatten", "flattens out code, removing nesting", createFlattenPass);
+  registerPass("extract-function-index",
+               "leaves just one function selected by index",
+               createExtractFunctionIndexPass),
```

Comment:
```suggestion
               createExtractFunctionIndexPass);
```
then the indentation below would be fixed by clang-format